src/deploy/NVA_build/standalone_deploy_nsfs.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+# Assumes that a postgresql is already running
+
+# This deployment script just automates the steps in the doc/standalone.md file
+
+function execute() {
+    if [[ -z "${CEPH_TEST_LOGS_DIR}" ]]; then
+        $1 &
+    else
+        $1 2>&1 | tee "${CEPH_TEST_LOGS_DIR}/${2}" &
+    fi
+}
+
+function main() {
+
+    # Add accounts to run ceph tests
+    execute "node src/cmd/manage_nsfs account add --config_root ./standalone/config_root --name cephalt --email ceph.alt@noobaa.com --new_buckets_path ./standalone/nsfs_root --access_key abcd --secret_key abcd" nsfs_cephalt.log
```

Comment:
Would you please set the constant separately?
For example: cephalt will be saved in the account name and when executed we will use the variable.

Reply:
I feel this way is okay. We are not going to achive any difference by setting storing the variables in constants and use them here. Since this is the only place we use them and those accounts wont change time to time. Keeping them as is will keep the things simple.